File: backend/aws-api/user-crud/GetController.py
import json

def get(connection):
    cursor = connection.cursor()
    cursor.execute("SELECT * FROM users")

    rows = cursor.fetchall()
    results = []
    for row in rows:
        results.append({
            "userId": row[0],
            "userName": row[1],
            "email": row[2],
            "membership": row[4]
        })
        
    return {
        'statusCode': 200,
        'headers': {
            'Content-Type': 'application/json',
            'Access-Control-Allow-Headers': 'Content-Type',
            'Access-Control-Allow-Origin': '*',
            'Access-Control-Allow-Methods': 'OPTIONS,GET'
        },
        'body': json.dumps(results)
    }

File: backend/aws-api/user-crud/GetController.js
exports.get = function(pool, callback) {
    const sql = "SELECT * FROM users";
    pool.getConnection(function(err, connection) {
        if(err) throw err;
        connection.query(sql, function(error, results) {
           if(error) {
               callback(error);
           } else {
               let r = [];
               for (var i = 0; i < results.length; i++) {
                   r.push({
                       userId: results[i]["user_id"],
                       userName: results[i]["user_name"],
                       email: results[i]["email"],
                       membership: results[i]["membership"]
                   });
               }
               const result = {
                   'statusCode': 200,
                    'headers': {
                        'Content-Type': 'application/json',
                        'Access-Control-Allow-Headers': 'Content-Type',
                        'Access-Control-Allow-Origin': '*',
                        'Access-Control-Allow-Methods': 'OPTIONS,GET'
                    },
                    'body': JSON.stringify(r)
               };
               callback(null, result);
           }
        });
    });
};